packages/schema/lib/schemas/OutputFieldsSchema.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const makeSchema = require('../utils/makeSchema');
+
+const OutputFieldSchema = require('./OutputFieldSchema');
+
+module.exports = makeSchema(
+  {
+    id: '/OutputFieldsSchema',
+    description: 'An array or collection of output fields.',
+    type: 'array',
+    items: { $ref: OutputFieldSchema.id },
```

Comment:
```suggestion
    items: {
      oneOf: [{ $ref: OutputFieldSchema.id }, { $ref: FunctionSchema.id }],
    },
```

Ditto https://github.com/zapier/zapier-platform/pull/957/files#r1932314176